projects/manager/src/app/datasets/dataset-editor/dataset-editor.component.html

```diff
@@ -94,5 +98,12 @@
                 <geoengine-manager-permissions resourceType="dataset" [resourceId]="dataset.id"></geoengine-manager-permissions>
             </mat-card-content>
         </mat-card>
+
+        <div class="actions">
+            <button mat-raised-button color="warn" *ngIf="!allowDeletion" (click)="allowDelete()">Delete Dataset</button>
+            <button mat-raised-button color="warn" *ngIf="allowDeletion" (click)="deleteDataset()">
+                Confirm Deletion (cannot be undone)
+            </button>
```

Comment:
This is odd. It changes the button. Haven't seen that before. Why not a model ok?

Reply:
Where do you have this from?
As I said, this seems uncommon.
I would expect a modal.

Reply:
f78fd74e9d422eb03857e29d7f49da61700cc1fe